app/core/views/_components/html.pug

```diff
@@ -9,9 +9,12 @@ mixin head()
     meta(http-equiv='X-UA-Compatible', content='IE=Edge')
     meta(content='width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no', name='viewport')
     link(href='https://fonts.googleapis.com/css?family=Lato' rel='stylesheet')
+    link(href='/css/home.min.css', rel='stylesheet')
```

Comment:
No need for this, the `build` logic generates a single `.css.min` file from `/source/css/`.

Reply:
I see. 
Now only 
link(href='/css/boilerplate.min.css', rel='stylesheet')